mmdet/core/bbox/iou_calculators/iou2d_calculator.py

```diff
@@ -0,0 +1,24 @@
+from ..geometry import bbox_overlaps
```

Comment:
There is only one method in `geometry.py`. We may consider moving it to this file.